### PR TITLE
get_items optimization

### LINKED ISSUE #224: news item sitemap query fires on every page
`WPSEO_News_Sitemap->get_items()` seems to be called every page view.

### PR DESCRIPTION
**Pull request by moorscode**
Instead of checking for items to add a filter on the `wpseo_sitemap_index` always add the filter and check inside if we need to modify anything.

Fixed several codestyle problems on this file.
Fixed codestyle config.
Updated composer yoast codestyle version.

Fixes #224

### CODE REVIEW

File: classes/class-sitemap.php [hunk 1]
```diff
@@ -240,8 +243,14 @@ private function get_post_types() {
 	}
 }
 
+/**
+ * Class WPSEO_News_Sitemap_Timezone
+ */
 class WPSEO_News_Sitemap_Timezone {
 
+	/**
+	 * @return string
```
Comment (andizer) [inline classes/class-sitemap.php]: Missing description

File: classes/class-sitemap.php [hunk 2]
```diff
@@ -304,7 +313,9 @@ private function get_timezone_id( $utc_offset ) {
 	}
 }
 
-
+/**
+ * Class WPSEO_News_Sitemap_Item
```
Comment (andizer) [inline classes/class-sitemap.php]: This isn't very descriptive.

File: classes/class-sitemap.php [hunk 3]
```diff
@@ -1,7 +1,14 @@
 <?php
+/**
+ * @package WPSEO\XML_Sitemaps
+ */
 
+/**
+ * Class WPSEO_News_Sitemap
```
Comment (andizer) [inline classes/class-sitemap.php]: This isn't very descriptive.

File: classes/class-sitemap.php [hunk 4]
```diff
@@ -240,8 +243,14 @@ private function get_post_types() {
 	}
 }
 
+/**
+ * Class WPSEO_News_Sitemap_Timezone
```
Comment (andizer) [inline classes/class-sitemap.php]: This isn't very descriptive.

File: classes/class-sitemap.php [hunk 5]
```diff
@@ -584,36 +592,28 @@ private function is_valid_datetime( $datetime ) {
 	}
 }
 
+/**
+ * Class WPSEO_News_Sitemap_Images
```
Comment (andizer) [inline classes/class-sitemap.php]: This isn't very descriptive.

File: classes/class-sitemap.php [hunk 6]
```diff
@@ -584,36 +592,28 @@ private function is_valid_datetime( $datetime ) {
...
-	 */
+	/** @var array Storage for the images */
 	private $images;
 
 	/**
 	 * Setting properties and build the item
 	 *
-	 * @param object $item
-	 * @param array  $options
+	 * @param object $item    This item.
```
Comment (andizer) [inline classes/class-sitemap.php]: This item?

**Comment by andizer** [comment]
CR done. Something failing with `__DIR__` in wpseo news.

**Comment by terw-dan** [comment]
CR2 done :+1:

**Comment by atimmer** [comment]
Acceptance done 👍🏻

**Pull request merged**